backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/lower/KonanDefaultParameterInjector.kt

```diff
@@ -29,6 +28,7 @@ internal class KonanDefaultParameterInjector(private val konanContext: KonanBack
             PrimitiveBinaryType.FLOAT -> IrConstImpl.float(startOffset, endOffset, type, 0.0F)
             PrimitiveBinaryType.DOUBLE -> IrConstImpl.double(startOffset, endOffset, type, 0.0)
             PrimitiveBinaryType.POINTER -> irCall(startOffset, endOffset, symbols.getNativeNullPtr.owner, emptyList())
+            PrimitiveBinaryType.VECTOR16 -> IrConstImpl.double(startOffset, endOffset, type, 0.0) // FIXME
```

Comment:
It makes sense to add a test for this. Something like
```kotlin
fun foo(v: NativeVector = vectorOf(1.0f, 2.0f, 3.0f, 4.0f)) = v
foo()
```
should cover this line.

Reply:
(Apparently `type` can't be `NativeVector` anymore since https://github.com/JetBrains/kotlin/commit/924fa6f9d3d16ef8e615ca639ba97d5646493aaa#diff-02427743f847bc86578ed06272b961f7R471, so we've decided to add such a test and throw an exception here).